src/main/java/it/gov/pagopa/payment/service/payment/common/CommonCreationServiceImpl.java

```diff
@@ -102,8 +101,7 @@ protected void checkInitiativeType(String initiativeId, InitiativeConfig initiat
               "[{}] Cannot find initiative with ID: [{}]",
               getFlow(),
               initiativeId);
-      throw new ClientExceptionWithBody(
-              HttpStatus.NOT_FOUND,
+      throw new InitiativeNotfoundException(
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor